study.md

```diff
@@ -246,5 +251,5 @@ Pick a few design heuristics. Describe some common, named refactorings you can
 use approach a good design.
 
 ```md
-<!-- your answer here -->
+My favorite is KISS. When I'm working on a codewars challenge, I can almost always tell when there's a method or technique that you're "supposed" to use, which I am ignorant of. My code goes beyond 5 lines, and doesn't make a whole lot of sense. This is my clue to do some googling. Quite often, some of the best solutions tend to be 3 lines.
 ```
```

Comment:
Great start. What would you use to DRY up duplicated logic? Maybe "Extract Method". Have a look at some named refactorings, like "Extract Method", to level up.